core/src/main/scala-3/sttp/tapir/macros/ValidatorMacros.scala

```diff
@@ -34,13 +34,13 @@ object ValidatorMacros {
         val instances = children.map(x => tpe.memberType(x).asType match { 
           case '[f] => Ref(x).asExprOf[f]
         })
-        val info = '{Option(SchemaType.SObjectInfo(${Expr(symbol.fullName)}))}
+        val name = '{Option(Schema.SName(${Expr(symbol.fullName)}))}
 
         '{
           Validator.Enumeration[T](
             List(${Varargs(instances)}: _*).asInstanceOf[List[T]],
             None,
-            ${info}
+            ${name}
```

Comment:
hmmm I'm not sure if `Validator` should contain an instance of `SName` which is defined in `Schema`. What do youI think about defining `SName` a separate file?

Reply:
Hm well that's the name of the schema. I think it belongs to `Schema`.